src/OrbitGl/ThreadStateBar.cpp

```diff
@@ -159,19 +173,36 @@ std::string ThreadStateBar::GetThreadStateSliceTooltip(PrimitiveAssembler& primi
 
   const auto* thread_state_slice =
       static_cast<const ThreadStateSliceInfo*>(user_data->custom_data_);
-  uint64_t begin_ns = thread_state_slice->begin_timestamp_ns();
-  uint64_t end_ns = thread_state_slice->end_timestamp_ns();
 
-  return absl::StrFormat(
+  std::string tooltip = absl::StrFormat(
       "<b>%s</b><br/>"
       "<i>Thread state</i><br/>"
       "<br/>"
-      "%s"
+      "%s",
+      GetThreadStateName(thread_state_slice->thread_state()),
+      GetThreadStateDescription(thread_state_slice->thread_state()));
+
+  if (thread_state_slice->wakeup_tid() != orbit_base::kInvalidThreadId) {
```

Comment:
What if `thread_state_slice->wakeup_tid() != orbit_base::kInvalidThreadId` but the reason is `kNotApplicable`?
I think we should check for both.
Alternatively, make the string returned by `GetWakeupReason` more generic, e.g., "created or unblocked".